deps/build.jl

```diff
@@ -1,3 +1,8 @@
+mkpath(joinpath(@__DIR__, "tzarchive"))
+mkpath(joinpath(@__DIR__, "compiled", string(VERSION)))
+touch(joinpath(@__DIR__, "active_version"))
+touch(joinpath(@__DIR__, "latest"))
```

Comment:
Do theses need to exist before using TimeZones? If so a comment mentioning that would be good